docs/examples.rst

```diff
@@ -1328,7 +1326,7 @@ Check the daylight limit.
     'EDT'
     >>> datetime(2003, 10, 26, 0, 59, tzinfo=tz).tzname()
     'EDT'
-    >>> datetime(2003, 10, 26, 1, 00, tzinfo=tz).tzname()
+    >>> datetime(2003, 10, 26, 2, 00, tzinfo=tz).tzname()
```

Comment:
I think this indicates we're not testing the doctests. Probably we should figure out how to mock out now in doctests so that we can actually get passing doctests.